lxl-web/src/hooks.server.ts

```diff
@@ -1,6 +1,16 @@
 import { defaultLocale, Locales } from '$lib/i18n/locales';
+import { USE_LOCAL_DISPLAY_JSONLD } from '$env/static/private';
+import { env } from '$env/dynamic/private';
+import { DisplayUtil, VocabUtil } from '$lib/utils/xl';
+import fs from 'fs';
+
+let utilCache;
 
 export const handle = async ({ event, resolve }) => {
+	const [vocabUtil, displayUtil] = await loadUtilCached();
+	event.locals.vocab = vocabUtil;
+	event.locals.display = displayUtil;
+
 	// set HTML lang
```

Comment:
Testing it locally seems to currently only result it one call to `loadUtil` so it probably isn't an issue for now – but it can be good to have it mind if multiple calls/requests are done further done the road.

Reply:
Writing it here for future reference: it can also be appropriate to use [<script context="module">](https://svelte.dev/docs/svelte-components#script-context-module) together with `+layout.server.ts` to initialize the utils class already at build time (but we also need have to add some functionality to update the initialized data whenever the vocab data is updated without having to re-build the entire app).

Reply:
:+1: 
Agreed. We should load vocab data in a robust way on startup (for example retrying until backend is up).  
We might want a way to reload it. Since it is a manual operation to update it (`fab <env> app.whelk.reload_syscore`) we have good control over when it happens.

It could be nice to have it update automatically when running with `USE_LOCAL_DISPLAY_JSONLD`